oci/oci.go

```diff
@@ -579,6 +545,7 @@ func waitContainerStop(ctx context.Context, c *Container, timeout time.Duration,
 			case <-chControl:
 				return
 			default:
+				// Check if the process is still around
```

Comment:
I think `FindProcess` and `ErrNotFound` below make this clear enough to not need this comment.  But it's obviously not a big deal if we want to add the comment either.